tests/jballerina-unit-test/src/test/resources/test-src/closures/closure.bal

```diff
@@ -751,3 +751,44 @@ function funcAcceptingIntArgs(int... scores) returns int {
     };
     return fn();
 }
+
+string moduleLevel = "module text";
+
+function errorConstructorReferenceTest1() returns error? {
+    string temp = "text";
+    var process = function() returns error? {
+        moduleLevel = "module text1";
+        return error("An Error", tempString = temp, moduleString = moduleLevel);
```

Comment:
Let's also use a param passed to errorConstructorReferenceTest1 here.

Reply:
Can we use `param1` here directly instead? Right now, as it is used in `temp`, we are still testing capturing `temp` only, rather than `param1` also.